compiler/passes/convert-typed-uast.cpp

```diff
@@ -4871,6 +4922,8 @@ bool TConverter::enter(const Call* node, RV& rv) {
     expr = x;
   } else if (auto x = convertNamedCallOrNull(node, rv)) {
     expr = x;
+  } else if (auto re = rv.byAst(node); re.type().isParam()) {
```

Comment:
Wait no way, is that how you write that pattern in C++? Holy crap! I've always wanted to do that but could never figure out how!